jobsapp/api/views/common.py

```diff
@@ -27,3 +34,18 @@ def get_queryset(self) -> "QuerySet[Job]":
             )
         else:
             return self.serializer_class.Meta.model.objects.filter(filled=False)
+
+
+class AboutUs(ListAPIView):
+    permission_classes = [AllowAny]
+
+    def get(self, request: Request, *args: Any, **kwargs: Any) -> Response:
+        about_content = FlatPage.objects.filter(url="/about-us/").first()
+        # REVIEW: Is this response format okay?
+        content = {
+            "url": reverse("about-us"),
+            "title": _(about_content.title),
+            # REVIEW: If there is any html tag we will send it, should we remove it before?
+            "content": _(about_content.content),
+        }
+        return Response(data=content, status=status.HTTP_200_OK)
```

Comment:
In my opinion, is not necessary to return the "url". And at the moment I don't know how manage translations for flatpages. 

At the moment, no return title and content with gettext.

Reply:
>  Django view?

Hey @francescarpi , there is already a django view for that. Is managed by flatpages it self. We can add or edit content on the admin page. I'm removing the translation part and I can take a look on _how translate flatpages_ later.

Reply:
Ok, thanks!